### DISCUSSION TITLE
Link preview on Discord

### DISCUSSION THREAD

**Discussion by KatyPurry** [bootscore/bootscore#218] [Ideas]
Hello,
If you put a link of my website on Discord (which is not a post or page, example the homepage), no image is inserted in the preview. Perhaps it would be a good idea to at least add the logo? 

Example : https://bootscore.me/2020/03/
Results : 
![image](https://user-images.githubusercontent.com/89610869/184997996-b6d20642-9817-42b6-8573-2bd161a2a709.png)

**Comment by justinkruit** [comment]
I highly recommend the usage of a plugin like Yoast or SeoPress to manage such things. If we start adding SEO stuff, the theme can easily become a whole lot more complicated.

**Reply by KatyPurry** [reply]
Yes, you're totally right !

**Comment by crftwrk** [comment]
This is an archive page, archives have no featured image. You should share a link to a post or page with featured image and Discord should show it (if Discord supports that).

For example: https://bootscore.me/blog/news/final-move-to-github/
